components/base/kinematicbase/fake_kinematics.go

```diff
@@ -36,7 +36,7 @@ func WrapWithFakeKinematics(
 		inputs:    []referenceframe.Input{{pt.X}, {pt.Y}, {0}},
 	}
 	var geometry spatialmath.Geometry
-	if fk.Base.Geometry != nil {
```

Comment:
suppose `fk.Base.Geometry := []spatialmath.Geometry{}`
here `fk.Base.Geometry != nil` is true but `fk.Base.Geometry[0]` does not exist.

ran into this when I ran a client script using a fake base.